python/lsst/eotest/sensor/MaskedCCD.py

```diff
@@ -276,7 +279,7 @@ def add_mask_files(mask_files, outfile, overwrite=True):
     for amp in masks:
         md = dafBase.PropertySet()
         md.set('EXTNAME', 'SEGMENT%s' % imutils.channelIds[amp])
-        masks[amp].writeFits(outfile, md, 'a')
+        masks[amp].writeFits(str(outfile), md, 'a')
```

Comment:
Is this cast really needed?

Reply:
Yes, I found that in some cases it was crashing b/c outfile was unicode.   I think that is was happening when outfile was generated starting from some pexConfig or argparse options.